services/apm/apm.js

```diff
@@ -47,7 +47,7 @@ class APMVersion extends BaseJsonService {
 
     const version = json.releases.latest;
     if (!version)
-      throw Error('Invalid version');
+      throw InvalidResponse('Invalid version');
```

Comment:
Hmm. You haven't actually imported `InvalidResponse` here. You want a call like: `const { InvalidResponse } = require('../errors');` as well.

I think this might be one of those javascript foot-guns where it looks like its is doing what you wanted but it is actually failing in a really subtly way because you can `throw` anything in js, including `undefined`, etc